aries-site/src/layouts/content/Subsection.js

```diff
@@ -22,22 +29,50 @@ export const Subsection = ({ children, name, ...rest }) => {
       onFocus={() => setOver(true)}
       onBlur={() => setOver(false)}
     >
-      <Box direction="row" justify="between">
-        <Subheading {...rest}>{name}</Subheading>
-        <Anchor
-          a11yTitle={`Jump to section titled ${name}`}
-          href={`#${id}`}
-          icon={<LinkIcon color={over ? 'text-xweak' : 'transparent'} />}
-        />
+      {/* This condition for gap is needed because the link icon has padding
+       * to maintain a big enough touch area for accessibility. This creates
+       * a larger than desired gap between h3's and its first child. This
+       * removes that extra space.
+       */}
+      <Box gap={GAP_SIZE[level]}>
```

Comment:
The context of `level` is different than `size`, level reminds me of Heading `level`, was it the purpose?

Reply:
Yes, the purpose is that Subsection receives a `level` (1, 2, 3, etc.) which will determine if the headings within that subsection are an h1, h2, h3, etc. 

The comment above line 37 is meant to explain why the gap needs to be adjusted based on the heading level. If it's not clear, I can try to reword.